### PR TITLE
Fixing the 2018 and 2019 solver description situation in SMT-COMP 2020

### PR DESCRIPTION
**Pull request by aehyvari**
The PR updates the available system descriptions for 2018 and fixes the names of 2019 solver descriptions according to what authors wrote in the description title.  The change only affects SMT-COMP 2020.

### CODE REVIEW

File: 2020/registration/solvers_divisions_final.csv [hunk 1]
```diff
@@ -34,30 +34,30 @@ Preliminary Solver ID,Solver ID,Wrapped Solver ID Single Query,Wrapped Solver ID
...
+23409,23703,24184,,,,2019-Vampire,https://vprover.github.io,vampire.pdf,Vampire 4.4-SMT System Description,no,UFDTNIA,,,,,Z3 4.5,,giles.reger@manchester.ac.uk,"Giles Reger, Martin Suda, Andrei Voronkov, Laura Kovacs, Evgenii Kotelnikov, Simon Robillard, Martin Riener",
 23470,23470,24192,24017,,24202,2019-Z3,,,,no,BVFP;FP;LIA;LRA;QF_IDL,BV;QF_UFLRA;UFLRA,,QF_ABV;QF_UFLRA,,,,,,
-23384,23730,24167,,,,2019-Poolector,https://boolector.github.io,boolector.pdf,Boolector,no,QF_BV,,,,23711,,,preiner@cs.stanford.edu,"Aina Niemetz, Mathias Preiner, Armin Biere",
-,19775,19775,,,,2018-CVC4,,,,no,ABVFP;AUFDTLIA;AUFLIA;QF_DT,,,,,,,,,
-,19783,19783,,,,2018-SMTRAT-Rat,,,,no,QF_NIRA,,,,,,,,,
+23384,23730,24167,,,,2019-Poolector,https://boolector.github.io,boolector.pdf,Boolector at the SMT Competition 2019,no,QF_BV,,,,23711,,,preiner@cs.stanford.edu,"Aina Niemetz, Mathias Preiner, Armin Biere",
+,19775,19775,,,,2018-CVC4,,CVC4.pdf,CVC4 at the SMT Competition 2018,no,ABVFP;AUFDTLIA;AUFLIA;QF_DT,,,,,,,,,
+,19783,19783,,,,2018-SMTRAT-Rat,,SMTRAT.pdf,SMT-RAT 2.2,no,QF_NIRA,,,,,,,,,
```
Comment (HanielB) [inline 2020/registration/solvers_divisions_final.csv:48]: These two lines have one column less than the original one, I think. Was this intentional?

File: 2020/registration/solvers_divisions_final.csv [hunk 2]
```diff
@@ -34,30 +34,30 @@ Preliminary Solver ID,Solver ID,Wrapped Solver ID Single Query,Wrapped Solver ID
...
+23416,23732,,,,24194,2019-CVC4-uc,http://cvc4.cs.stanford.edu,cvc4.pdf,CVC4 at the SMT Competition 2019,no,,,,AUFNIRA;BV;UFIDL,"CVC4, 23420",,,noetzli@cs.stanford.edu,"Clark Barrett, Haniel Barbosa, Martin Brain, Tim King, Makai Mann, Aina Niemetz, Andres Nötzli, Alex Ozdemir, Mathias Preiner, Andrew Reynolds, Cesare Tinelli, Yoni Zohar",
+23427,23679,24162,23971,,24196,2019-MathSAT-na-ext,,mathsat5.pdf,MathSAT5 (Nonlinear) at the SMT Competition 2019,no,,QF_AUFBVNIA,,QF_NIRA,23680,,,airfan@stanford.edu,"Alberto Griggio, Ahmed Irfan, Alessandro Cimatti, Roberto Sebastiani",
+23364,23695,,23951,,,2019-Yices 2.6.2 Incremental,http://yices.csl.sri.com,yices.pdf,Yices 2 in SMT-COMP 2019,no,,QF_AUFBV;QF_BV;QF_UF,,,23694,,,bruno.dutertre@sri.com,"Bruno Dutertre, Dejan Jovanovic, Ian A. Mason, Stéphane Graham-Lengrand",
+,19991,,19991,,,2018-Boolector (incremental),,Boolector.pdf,Boolector at the SMT Competition 2018,no,,QF_ABV;QF_UFBV,,,,,,,,
+,19992,,19992,,,2018-CVC4 (incremental),,CVC4.pdf,CVC4 at the SMT Competition 2018,no,,ANIA,,,,,,,,
```
Comment (HanielB) [inline 2020/registration/solvers_divisions_final.csv:57]: ditto

File: 2020/registration/solvers_divisions_final.csv [hunk 3]
```diff
@@ -34,30 +34,30 @@ Preliminary Solver ID,Solver ID,Wrapped Solver ID Single Query,Wrapped Solver ID
...
 ,19993,,19993,,,2018-MathSAT (incremental),,,,no,,QF_LRA,,,,,,,,
 ,19995,,19995,,,2018-Yices (incremental),,,,no,,QF_AUFLIA;QF_LIA,,,,,,,,
 ,19996,,19996,,,2018-Z3 (incremental),,,,no,,ALIA;LIA;QF_ALIA;QF_ANIA;QF_UFLIA,,,,,,,,
-,19793,,,,19793,2018-CVC4 (unsat core),,,,no,,,,AUFLIA;AUFLIRA;QF_ANIA;QF_AUFLIA;QF_NRA;UF;UFIDL;UFLIA;UFNIA,,,,,,
+,19793,,,,19793,2018-CVC4 (unsat core),,CVC4.pdf,CVC4 at the SMT Competition 2018,no,,,,AUFLIA;AUFLIRA;QF_ANIA;QF_AUFLIA;QF_NRA;UF;UFIDL;UFLIA;UFNIA,,,,,,
```
Comment (HanielB) [inline 2020/registration/solvers_divisions_final.csv:61]: ditto

File: 2020/registration/solvers_divisions_final.csv (hunk 1 above)
Comment (aehyvari) [reply 2020/registration/solvers_divisions_final.csv:48]: No, certainly not intentional.  However, given that's it really hard to count these, I tried to automate a little, and I couldn't confirm your observation!

Here's what I did:
```
$ git checkout origin/solver-descriptions
HEAD is now at e57889e Fixing the 2018 and 2019 solver description situation
$ cat solvers_divisions_final.csv |while read -r line; do echo ${line} |csvcut -n |wc -l; done |sort |uniq
      21
$ git checkout origin/master
Previous HEAD position was e57889e Fixing the 2018 and 2019 solver description situation
HEAD is now at 9945985 A suggestion for disabling competition-wide winner reporting for a track
$ cat solvers_divisions_final.csv |while read -r line; do echo ${line} |csvcut -n |wc -l; done |sort |uniq
      21
```

So unless there's something wrong with this methodology, it looks like in both versions `solvers_divisions_final.csv` has the same amount of fields.

Comment (aehyvari) [reply 2020/registration/solvers_divisions_final.csv:48]: Anyway, it was good to check this, thanks.  I'm resolving the conversation.

File: 2020/registration/solvers_divisions_final.csv (hunk 2 above)
Comment (aehyvari) [reply 2020/registration/solvers_divisions_final.csv:57]: See above.

File: 2020/registration/solvers_divisions_final.csv (hunk 3 above)
Comment (aehyvari) [reply 2020/registration/solvers_divisions_final.csv:61]: see above

File: 2020/registration/solvers_divisions_final.csv (hunk 1 above)
Comment (HanielB) [reply 2020/registration/solvers_divisions_final.csv:48]: Yeah, I miscounted :) Sorry and thanks for checking carefully.

**Comment by jhoenicke** [comment]
I think the system description URLs point to the wrong directory.  I guess you would need to use the full URL`https://smtcomp.github.io/2019/system_descriptions/` in front of the pdf.  Or add `../../2019/system_descriptions`, but then the URL looks ugly.

**Comment by aehyvari** [comment]
> I think the system description URLs point to the wrong directory. I guess you would need to use the full URL`https://smtcomp.github.io/2019/system_descriptions/` in front of the pdf. Or add `../../2019/system_descriptions`, but then the URL looks ugly.

I just created [this pull request](https://github.com/SMT-COMP/smt-comp.github.io/pull/7) for the web site that suggests how to address this issue.

**Pull request merged**